### ISSUE TITLE
Checksum issue

### ISSUE THREAD

**Issue by damianoneill** [andrewrothstein/ansible-kubectl#9]
Hi @andrewrothstein 

I was for updating the releases to include support for v1.17.5, I used your dl-checksum.sh script to generate the shas and got the following:

```console
✔ ~/projects/github.com/damianoneill/ansible-kubectl [master|✔]
13:55 $ ./dl-checksum.sh v1.17.5
  v1.17.5:
    linux:
      # https://storage.googleapis.com/kubernetes-release/release/v1.17.5/bin/linux/amd64/kubectl
      amd64: sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
      # https://storage.googleapis.com/kubernetes-release/release/v1.17.5/bin/linux/arm64/kubectl
      arm64: sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
    darwin:
      # https://storage.googleapis.com/kubernetes-release/release/v1.17.5/bin/darwin/amd64/kubectl
      amd64: sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
    windows:
      # https://storage.googleapis.com/kubernetes-release/release/v1.17.5/bin/windows/amd64/kubectl.exe
      amd64: sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
```

Note all the SHAs above are the same! 

When I use this version I get the following output error 

```console
TASK [andrewrothstein.kubectl : downloading https://storage.googleapis.com/kubernetes-release/release/v1.17.5/bin/linux/amd64/kubectl to /usr/local/bin/kubectl-v1.17.5...] ********
fatal: [qnc-davinci-10]: FAILED! => {"changed": true, "checksum_dest": null, "checksum_src": "b115f01217e22b206bcc37d71cc1a1da33f7cf5e", "dest": "/usr/local/bin/kubectl-v1.17.5", "elapsed": 1, "msg": "The checksum for /usr/local/bin/kubectl-v1.17.5 did not match e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855; it was 03cd1fa19f90d38005148793efdb17a9b58d01dedea641a8496b9cf228db3ab4.", "src": "/tmp/ansible-moduletmp-1612360937.78-a9xjhP/tmpW94k4z", "url": "https://storage.googleapis.com/kubernetes-release/release/v1.17.5/bin/linux/amd64/kubectl"}
```

Any ideas why this is failing?

Thanks,
Damian.

**Comment by andrewrothstein** [comment]
not sure why the bash script wasn't working for you. I added checksums for v1.17.5 and published to the galaxy as v1.1.54

**Comment by damianoneill** [comment]
Thanks Andrew.
